.golangci.yaml

```diff
@@ -75,14 +131,10 @@ linters:
     - typecheck
     - unconvert
     - unused
-    - varcheck
     - whitespace
     - bodyclose
     - noctx
     - rowserrcheck
-    - structcheck
-    - unparam
-
```

Comment:
Are these removals intentional?

Reply:
Yes, these are deprecated, updated the PR description

Reply:
Few of the linters need to address separately as many files were flaged with lint issues.
Added TODO in the golangci.yaml